manifests/master/tuning.pp

```diff
@@ -26,26 +26,24 @@
 
     case $jvm_tuning_profile {
       'lvm': {
-        $amq_heap_mb                = '96'
```

Comment:
I meant to change the LVM profile.  I want to figure out what the reasonable minimum is in that profile and use that for this case.